### PR TITLE
feat: update edit tile content modal

### PR DESCRIPTION
**Pull request by yasmin-e**
Closes: #6150

### CODE REVIEW

File: packages/frontend/src/components/DashboardTiles/TileForms/ChartUpdateModal.tsx [hunk 1]
```diff
@@ -0,0 +1,125 @@
...
+
+const ChartUpdateModal = <T extends DashboardTile>({
+    tile,
+    onClose,
+    onConfirm,
+    ...modalProps
+}: ChartUpdateModalProps<T>) => {
+    const form = useForm({
+        initialValues: {
+            uuid: tile.properties.savedChartUuid,
```
Comment (yasmin-e) [inline packages/frontend/src/components/DashboardTiles/TileForms/ChartUpdateModal.tsx:35]: TS error here:
`TS2339: Property 'savedChartUuid' does not exist on type '{ title: string | null; hideTitle?: boolean | undefined; savedChartUuid: string | null; } | { title: string; hideTitle?: boolean | undefined; url: string; } | { title: string; content: string; }'.   Property 'savedChartUuid' does not exist on type '{ title: string; hideTitle?: boolean | undefined; url: string; }'.`

File: packages/frontend/src/components/DashboardTiles/TileBase/index.tsx [hunk 2]
```diff
@@ -192,18 +193,29 @@ const TileBase = <T extends Dashboard['tiles'][number]>({
...
+                            isOpen={isReplacingChart}
+                            tile={tile}
+                            onClose={() => setIsReplacingChart(false)}
+                            onConfirm={(newTile) => {
+                                onEdit(newTile);
+                                setIsReplacingChart(false);
+                            }}
+                        />
+                    )}
                     <TileUpdateChartTitle
```
Comment (yasmin-e) [inline packages/frontend/src/components/DashboardTiles/TileBase/index.tsx:219]: @rephus I noticed the title is not updating when I update a tile 🤔  any thoughts on how to fix this?

Comment (rephus) [reply packages/frontend/src/components/DashboardTiles/TileBase/index.tsx:219]: mm, the title is passed as a property on DashboardChartTileMain and TileBase, if the chart is different, the title should be different too, :thinking:

**Comment by rephus** [comment]
I can't get render to deploy, there is a type error in the code :

> Jul 12 09:03:20 AM  #35 23.15 src/components/DashboardTiles/TileForms/ChartUpdateModal.tsx(35,35): error TS2339: Property 'savedChartUuid' does not exist on type '{ title?: string | undefined; hideTitle?: boolean | undefined; savedChartUuid: string | null; } | { title: string; content: string; } | { title: string; hideTitle?: boolean | undefined; url: string; }'.
Jul 12 09:03:20 AM  #35 23.15   Property 'savedChartUuid' does not exist on type '{ title: string; content: string; }'.
Jul 12 09:03:20 AM  #35 23.15 src/components/DashboardTiles/TileForms/ChartUpdateModal.tsx(45,34): error TS2339: Property 'savedChartUuid' does not exist on type 'WritableDraft<{ title?: string | undefined; hideTitle?: boolean | undefined; savedChartUuid: string | null; }> | WritableDraft<{ title: string; content: string; }> | WritableDraft<...>'.
Jul 12 09:03:20 AM  #35 23.15   Property 'savedChartUuid' does not exist on type 'WritableDraft<{ title: string; content: string; }>'.

**Comment by yasmin-e** [comment]
> I can't get render to deploy, there is a type error in the code :

fixed the error, should be available soon @rephus

**Pull request closed without merging**